WooCommerce/Resources/en.lproj/Localizable.strings

```diff
@@ -1780,6 +1792,9 @@
 /* Error message on the Coupon Details screen when deleting coupon fails */
 "Failed to delete coupon. Please try again." = "Failed to delete coupon. Please try again.";
 
+/* Error message on the Issue Refund screen when fetching the charge details failed */
+"Failed to fetch your charge details. Please try again." = "Failed to fetch your charge details. Please try again.";
```

Comment:
Found in the version with the new tooling, too. See [`5c21528`](https://github.com/woocommerce/woocommerce-ios/pull/6737/commits/5c215284d6839f397732f684ec664405f42939ab#diff-96d03942a96e588fcdca0288a2a18d0466a6e44ee5b256f581cd17f2086ea22dR1880-R1881)